src/pulse3D/metrics.py

```diff
@@ -439,14 +437,20 @@ def calculate_twitch_velocity(self, coordinate_df: DataFrame, is_contraction: bo
         Returns:
             DataFrame floats that are the velocities of each twitch
         """
-        coord_type = "contraction" if is_contraction else "relaxation"
-
-        Y_end = coordinate_df["force", coord_type, self.velocity_start]
-        Y_start = coordinate_df["force", coord_type, self.velocity_end]
+        if is_contraction:
+            coord_type = "contraction"
+            twitch_base = self.baseline_widths[0]
+            twitch_top = 90
```

Comment:
Should the twitch top be c100 and r0?

Reply:
yeah that's what I was going to ask Jason tomorrow, wasn't sure? and we will need to prevent users from inputting whatever max and min widths here because that will cause the same problem

Reply:
C100 and R0 seem right

Reply:
It's definitely possible though that 10 and 90 were the bounds for both prior to this change

Reply:
<img width="268" alt="Screenshot 2023-05-20 at 6 33 00 AM" src="https://github.com/CuriBio/pulse3d/assets/84344924/d3d43eb6-618d-4798-aaf6-86db99572c3f">
it actually does need to be R10 and C90